src/test/java/de/retest/web/util/SeleniumWrapperUtilTest.java

```diff
@@ -66,4 +66,28 @@ void should_get_wrapped_driver() throws Exception {
 		assertThat( SeleniumWrapperUtil.getWrapped( WrapperOf.DRIVER, oldWrapper ) ).isSameAs( wrapped );
 	}
 
+	@Test
+	void getWrapped_should_not_silently_ignore_exceptions_thrown_by_method() {
+		final WrapsElement elementThrows = mock( WrapsElement.class );
+		when( elementThrows.getWrappedElement() ).thenThrow( new NoSuchElementException( "No element found" ) );
+
+		assertThatCode( () -> SeleniumWrapperUtil.getWrapped( WrapperOf.ELEMENT, elementThrows ) )
```

Comment:
Do you know what is the difference to `assertThatThrownBy`?

Reply:
Firstly, I would have said nothing, since they have the same return. But looking at the code `assertThatThrownBy` verifies that an exception has been thrown. That means that `assertThatCode` can expect no assertion thrown.

Since I expect a exception, I will change it

Reply:
Didn't know this too, thx for looking it up. 👍